src/cli/manage.rs

```diff
@@ -44,6 +44,13 @@ pub struct Manage {
     /// The configuration source for a management server.
     #[clap(subcommand)]
     pub provider: crate::config::Providers,
+    /// If specified, filters the available gameserver addresses to the one that
+    /// matches the specified type
+    #[clap(long)]
+    pub address_type: Option<String>,
+    /// If specified, additionally filters the gameserver address by its ip kind
+    #[clap(long, requires("address_type"), value_enum)]
+    pub ip_kind: Option<crate::config::AddrKind>,
```

Comment:
You can use `default_value_t` here so that you don't need to have the `unwrap_or`.
```suggestion
    #[clap(long, requires("address_type"), value_enum, default_value_t=crate::config::AddrKind::Any)]
    pub ip_kind: crate::config::AddrKind,
```